torch/ao/quantization/_dbr/auto_trace_rewriter.py

```diff
@@ -100,6 +100,14 @@ def create_node(self, kind, target, args, kwargs, name=None, type_expr=None):
         if target == operator.mul:
             target = torch.mul
 
+        # TODO(future PR): move this into mappings
+        if target == 'add':
```

Comment:
guess it's OK for now. in the long term would you prefer to use torch.Tensor.add or `add`?

Reply:
I think `torch.Tensor.add` sgtm for user facing APIs.  This codepath has to match the FX representation, so this codepath probably wouldn't change.